site/src/docs/components/highlight/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/components/highlight'
+title: 'Highlight'
+navTitle: 'Highlight'
+---
+
+import { Highlight } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview style={{ backgroundColor: 'var(--color-black-5)' }}>
```

Comment:
Just curious, why the background color in these Playgrounds? I think these look better without it.

Reply:
Bump up reminderino